numpy/ma/core.py

```diff
@@ -5640,6 +5640,12 @@ def ptp(self, axis=None, out=None, fill_value=None):
         np.subtract(out, min_value, out=out, casting='unsafe')
         return out
 
+    def partition(self, *args, **kwargs):
+        warnings.warn("Warning: partition will ignore masked items "
```

Comment:
"ignore the mask" is different to "ignore masked items". The former is what happens, the latter is what the user expects.